projects/Mallard/src/screens/article/slider/SliderHeaderLowEnd.tsx

```diff
@@ -73,7 +73,14 @@ const SliderHeaderLowEnd = withNavigation(
                 >
                     <SliderBarWrapper
                         sections={sections}
-                        sliderPosition={sliderPosition}
+                        sliderPosition={
+                            Platform.OS === 'android'
+                                ? sliderPosition
+                                : Animated.divide(
+                                      sliderPosition,
+                                      new Animated.Value(width),
```

Comment:
Question (and I don't know the answer). Is there a cost to object creation here? Can it not be assigned to a variable and used only once?

Reply:
Quite possibly a small one. Will continue to look into perf.